reflex/app.py

```diff
@@ -128,9 +128,10 @@ def __init__(self, *args, **kwargs):
 
         # Create the socket app. Note event endpoint constant replaces the default 'socket.io' path.
         self.socket_app = ASGIApp(self.sio, socketio_path="")
-
+        namespace = config.get_event_namespace()
+        assert namespace
```

Comment:
`assert` is ignored when running with `python -O` and should only be used in test code. This check should raise an exception instead.

```python
if not namespace:
    raise RuntimeError("event_namespace must be provided in the config")
```